sdk/provisioning/Azure.Provisioning.AppConfiguration/tests/BasicAppConfigurationTests.cs

```diff
@@ -85,7 +77,7 @@ await test.Define(
             }
 
             resource featureFlag 'Microsoft.AppConfiguration/configurationStores/keyValues@2022-05-01' = {
-                name: '.appconfig.featureflag~2FFeatureFlagSample'
+                name: '.appconfig.featureflag~2F${featureFlagKey}'
```

Comment:
How did `{featureFlagKey}` get injected into here?

Reply:
The string interpolation for parameters bug fix added the `${...}`.  The definition looked like: https://github.com/Azure/azure-sdk-for-net/blob/aae6107fb8af07de62cb73fc96ae4f5d04609eab/sdk/provisioning/Azure.Provisioning.AppConfiguration/tests/BasicAppConfigurationTests.cs#L52